client/src/controllers/FormsetController.ts

```diff
@@ -41,12 +45,15 @@ export class FormsetController extends Controller<HTMLElement> {
   static classes = ['deleted'];
 
   static targets = [
+    'add',
     'child',
+    'delete',
     'deleted',
     'deleteInput',
     'forms',
-    'minFormsInput',
     'maxFormsInput',
+    'minFormsInput',
```

Comment:
I know this re-order was in one of the earlier commits, but it's probably not necessary, feel free to keep the `minFormsInput` before the `maxFormsInput`.